Makefile

```diff
@@ -13,6 +13,8 @@ build:
 	go build -o $(BUILD_ARCH)/$(BIN_DIR)/dp-filter-api cmd/$(MAIN)/main.go
 debug: build
 	HUMAN_LOG=1 go run -race cmd/$(MAIN)/main.go
+acceptance: build
```

Comment:
this is probably a pattern we should use elsewhere (I've been bitten by this, today, on another PR!)

is this documented/used anywhere that you can point to (README should mention it?)?

Reply:
I thrown a section in the readme to cover running the acceptance tests